en/docs/includes/db-advanced-config.md

```diff
@@ -24,5 +25,6 @@ Apart from the basic configurations specified above, WSO2 Identity Server suppor
 	validationQuery="SELECT 1"
 	validationInterval="30000"
 	defaultAutoCommit=false
+    commitOnReturn=true
```

Comment:
Again, as said previously, there is a slight indentation issue.

Reply:
Fixed with https://github.com/wso2/docs-is/commit/ddfc78d45eacc188cbc0a192bec916591e7b78ae.